### ISSUE TITLE
Suggestion - Spreadsheet

### ISSUE THREAD

**Issue by PatrickJnr** [Nukem9/dlssg-to-fsr3#72]
I think it'd be a good idea if we have a spreadsheet of what games currently work. 

These games work for me.

Starfield
Cyberpunk
Hitman 3

**Comment by FlowersPowerz** [comment]
immortals of aveum works and is much better than native implementation

**Comment by XXXBIOFOXXX** [comment]
The Witcher 3 work fine, i have rtx 3060 8gb, 32 ram y ryzen 5600, 1440p, have 90 fps whitout rtx and Ultra.

**Comment by Nooblet16** [comment]
Pretty much all UE5 games that have DLSS framegen work perfectly with Nukem's nvgx.dll mod. LukeFZ's mod has issues with UI on the other hand in UE5 games, the same kind of UI issues you see in Cyberpunk and Darktide.

**Comment by FlowersPowerz** [comment]
> Pretty much all UE5 games that have DLSS framegen work perfectly with Nukem's nvgx.dll mod. LukeFZ's mod which is based on winmm.dll has issues with UI on the other hand in UE5 games, the same kind of UI issues you see in Cyberpunk and Darktide with Nukem's mod. From what I see LukeFZ's mod gets rid of ghosting in Cyberpunk.
> 
> Games I can confirm that work:
> 
> (No issues) Remnant 2 Robocop Fort Solis Immortals of Aveum
> 
> Work with UI issues and/or some ghosting:
> 
> Darktide (Had some crashes but not sure if due to Ray Tracing, RTSS or framegen) Cyberpunk

There aren't ghosting or ui distortion in the latest version of the mod (0.41)

**Comment by Nukem9** [comment]
> I think it'd be a good idea if we have a spreadsheet of what games currently work.
> 
> These games work for me.
> 
> Starfield Cyberpunk Hitman 3

Yeah, I'll see about it in a day or two. I'd just rather spend my time writing code instead of managing issues.

**Comment by alex-enginity** [comment]
Alan Wake 2 works perfectly on my RTX3090 Ti

**Comment by FlowersPowerz** [comment]
> > immortals of aveum works and is much better than native implementation
> 
> How did you enable it? Where do I put the files, and what needs to be done in the game menu?

steamapps\common\ImmortalsOfAveum\x64\Windows\P3\Binaries\Win64

put the files here, and active in the menu frame generation with dlss at your preferred quality

**Comment by Nooblet16** [comment]
> > Pretty much all UE5 games that have DLSS framegen work perfectly with Nukem's nvgx.dll mod. LukeFZ's mod which is based on winmm.dll has issues with UI on the other hand in UE5 games, the same kind of UI issues you see in Cyberpunk and Darktide with Nukem's mod. From what I see LukeFZ's mod gets rid of ghosting in Cyberpunk.
> > Games I can confirm that work:
> > (No issues) Remnant 2 Robocop Fort Solis Immortals of Aveum
> > Work with UI issues and/or some ghosting:
> > Darktide (Had some crashes but not sure if due to Ray Tracing, RTSS or framegen) Cyberpunk
> 
> There aren't ghosting or ui distortion in the latest version of the mod (0.41)

There is on Darktide.

**Comment by MJ12MJ** [comment]
Works on Hogwarts Legacy too, but tested with version 0.41, I don't know about the latest 0.50 (HDR should be tested too, I don't have it).

**Comment by cichy45** [comment]
I created PR with game list, if you test that some game works please comment with below schema, so it can be included.
https://github.com/Nukem9/dlssg-to-fsr3/pull/85

| Game                        | Game Version | Mod version | Mod install | FrameGen Works | Ingame HDR | Other issues                                                         |
|-----------------------------|---------|-------------|-------------|-------|------------|----------------------------------------------------------------------|
| Cyberpunk 2077              | 2.1     | 0.5         | DLSSTweaks ini method  | ✅     | ✅          | Ghosting/artifact line at the bottem of the screen when driving fast |

**Comment by Jackich2** [comment]
> Works on Hogwarts Legacy too, but tested with version 0.41, I don't know about the latest 0.50 (HDR should be tested too, I don't have it). RTX 3080

t does not work for me FG, lit gray.. where did you put the files?

**Comment by MJ12MJ** [comment]
> > Works on Hogwarts Legacy too, but tested with version 0.41, I don't know about the latest 0.50 (HDR should be tested too, I don't have it). RTX 3080
> 
> t does not work for me FG, lit gray.. where did you put the files?

Here
https://ibb.co/4PxjRVk

**Comment by Daineli** [comment]
Slender: The Arrival works for me, Testing 0.41 with normal method. Screens with and without mod. Not tested 0.5 yet, I think it should works as before.
Witcher 3 works with normal version.
![SlenderTheArrival-Win64-Shipping_2023_12_18_22_27_53_281](https://github.com/Nukem9/dlssg-to-fsr3/assets/154284902/58333d37-5f57-42b6-b531-4095e4e29ed6)
![SlenderTheArrival-Win64-Shipping_2023_12_18_22_27_28_363](https://github.com/Nukem9/dlssg-to-fsr3/assets/154284902/47649f38-851b-446f-bf6c-aa07ae3c51dc)

**Comment by cichy45** [comment]
> t does not work for me FG, lit gray.. where did you put the files?

If you get the information popup when starting game but frame gen is grayed out in settings, make sure you have hardware gpu scheduling enabled in windows.

**Comment by Daineli** [comment]
Deliver Us Mars, epic version, Work only when I run from exe, not working from Epic launcher.
Tested dlss tweaks version 0.5.

Looks work, there is probably shimering, but the most I see shimering there is hair when I use mouse left to right so fast, but with normal like gamepad this is almost not visible. 
Screens on off, and option where you can find FG - sorry for non english version game.
![DeliverUsMars-Win64-Shipping_2023_12_19_15_57_13_124](https://github.com/Nukem9/dlssg-to-fsr3/assets/154284902/47b39d1f-dfa8-4747-93bf-d68fb452c4ec)
![DeliverUsMars-Win64-Shipping_2023_12_19_15_50_40_903](https://github.com/Nukem9/dlssg-to-fsr3/assets/154284902/c54ffe57-ec1e-4fa9-84b5-74b1c66bebe7)
![DeliverUsMars-Win64-Shipping_2023_12_19_15_57_28_649](https://github.com/Nukem9/dlssg-to-fsr3/assets/154284902/419437e4-46ce-411e-8644-1405a64f3e7b)

**Comment by ejams1** [comment]
Jedi survivor works with HDR on 0.5.
> User had to rename nvngx.dll to dxgi.dll to make it work
This is only required if the registry tweak is not applied, and is likely the case for all games

**Comment by slyfox8900** [comment]
> Jedi survivor works with HDR on 0.5.
> 
> > User had to rename nvngx.dll to dxgi.dll to make it work
> 
> This is only required if the registry tweak is not applied, and is likely the case for all games

What folders did you put the files in? Did you use DLSS tweaks? Jedi survivor as well as hogwarts legacy dont have the DLSS dll files next to the games exe, they are in their own folders and their are 2 nvidia folders with Dlss dlls in them. I can't seem to get this mod to work no matter where i put the mod files.

**Comment by ejams1** [comment]
@slyfox8900 the docs for DLSSTweaks discuss this more specifically as unreal engine based games like Jedi and hogwarts actually have two EXEs - the one at the root of the game folder (a few hundred KB) and the real one in the bin folder (few hundred MB). So put the files next to the real one, not the smaller shortcut one. I use DLSSTweaks as it lets you explicitly tell it which DLL file to load which in this case is the wrapper

**Comment by slyfox8900** [comment]
> @slyfox8900 the docs for DLSSTweaks discuss this more specifically as unreal engine based games like Jedi and hogwarts actually have two EXEs - the one at the root of the game folder (a few hundred KB) and the real one in the bin folder (few hundred MB). So put the files next to the real one, not the smaller shortcut one. I use DLSSTweaks as it lets you explicitly tell it which DLL file to load which in this case is the wrapper

So i got jedi survivor to work without dlss tweaks. But Hogwarts refuses to hook the mod for me either way. What version of the game are you using? Should show up on the title menu

**Comment by ejams1** [comment]
I'm using the latest version as steam keeps it up to date. Hogwarts should work the same although I've been using DLSSTweaks for all of them and they all work that way

**Comment by FlowersPowerz** [comment]
Tried the empress version of Hogwarts legacy and the mod not work, but with the original steam version works but if I move the slider for clarity it crash. Seems to be incompatible for now witch cracked version of the game.

**Comment by PatrickJnr** [comment]
> > I think it'd be a good idea if we have a spreadsheet of what games currently work.
> > These games work for me.
> > Starfield Cyberpunk Hitman 3
> 
> Yeah, I'll see about it in a day or two. I'd just rather spend my time writing code instead of managing issues.

Thats fair and all, I meant as a community effort. Amazing work though :D

**Comment by kedaitinh12** [comment]
For LukeFZ 0.7.2 you can use here
https://drive.google.com/file/d/1-vgQPeIk_7wpgjiqqiqmoQUlrvDi0X6F/view?usp=drivesdk

And read here to know what file to use. Read redme in file download to know how to mod
https://docs.google.com/spreadsheets/d/1bGDjgkRahwNvD2XtqiHdZBRc6VB52UKsr1Qn1TmvSHo/edit#gid=0

**Comment by alex-enginity** [comment]
Thank you!

**Comment by remileduc** [comment]
> Deliver Us Mars, epic version, Work only when I run from exe, not working from Epic launcher. Tested dlss tweaks version 0.5.

I can't make it work... I'm trying with 0.81, and I tried putting the files in different folders, but it doesn't seem to work:
- main folder where the main EXE is
- `DeliverUsMars\Binaries\Win64` where the EXE *DeliverUsMars-Win64-Shipping.exe* is
- `DeliverUsMars\Plugins\DLSS\Binaries\ThirdParty\Win64` where the DLSS dll is
- `DeliverUsMars\Plugins\Streamline\Binaries\ThirdParty\Win64` where the DLSS FG dll is

**Comment by Daineli** [comment]
> @Daineli
> 
> > Deliver Us Mars, epic version, Work only when I run from exe, not working from Epic launcher. Tested dlss tweaks version 0.5.
> 
> I can't make it work...
> 
> _EDIT_ It _is_ working with DLSS Tweaks, but you have to do it this way: Install the files in `DeliverUsMars\Binaries\Win64` where _DeliverUsMars-Win64-Shipping.exe_ is and change the DLSS dlls paths in the _.ini_.
> 
> Start the game from `DeliverUsMars\Binaries\Win64\DeliverUsMars-Win64-Shipping.exe` (and not from the main exe, not from EGS).

Yep. only version dlss tweak- every dlss tweaks should use with edited in .ini as wrote in readme.
Exe- using the same from folder where we put files, as You've described

**Comment by kedaitinh12** [comment]
> @Daineli
> 
> > Deliver Us Mars, epic version, Work only when I run from exe, not working from Epic launcher. Tested dlss tweaks version 0.5.
> 
> I can't make it work...
> 
> _EDIT_ It _is_ working with DLSS Tweaks, but you have to do it this way: Install the files in `DeliverUsMars\Binaries\Win64` where _DeliverUsMars-Win64-Shipping.exe_ is and change the DLSS dlls paths in the _.ini_.
> 
> Start the game from `DeliverUsMars\Binaries\Win64\DeliverUsMars-Win64-Shipping.exe` (and not from the main exe, not from EGS).

*
To use mod with Epic ver: you have to remove/delete all files in those two folders:

C:\Program Files (x86)\Epic Games\Launcher\Portal\Extras\Overlay
C:\Program Files (x86)\Epic Games\Epic Online Services\managedArtifacts\98bc04bc842e4906993fd6d6644ffb8d

**Comment by GFR3-404NOtfOuND** [comment]
> @Daineli
> 
> > Deliver Us Mars, epic version, Work only when I run from exe, not working from Epic launcher. Tested dlss tweaks version 0.5.
> 
> I can't make it work...
> 
> _EDIT_ It _is_ working with DLSS Tweaks, but you have to do it this way: Install the files in `DeliverUsMars\Binaries\Win64` where _DeliverUsMars-Win64-Shipping.exe_ is and change the DLSS dlls paths in the _.ini_.
> 
> Start the game from `DeliverUsMars\Binaries\Win64\DeliverUsMars-Win64-Shipping.exe` (and not from the main exe, not from EGS).

Hi, do I have the right files? I have no frame gen and DLSS option in the settings menu, only TAAU and FSR. I also enabled nvidia signature override.
